Source/Particles/Gather/GetExternalFields.cpp

```diff
@@ -29,6 +29,21 @@ GetExternalEField::GetExternalEField (const WarpXParIter& a_pti, int a_offset) n
         m_yfield_partparser = getParser(mypc.m_Ey_particle_parser);
         m_zfield_partparser = getParser(mypc.m_Ez_particle_parser);
     }
+    else if (mypc.m_E_ext_particle_s=="repeated_plasma_lens")
+    {
+        m_type = RepeatedPlasmaLens;
+        m_dt = warpx.getdt(a_pti.GetLevel());
+        m_get_position = GetParticlePosition(a_pti, a_offset);
+        auto& attribs = a_pti.GetAttribs();
+        m_ux = attribs[PIdx::uy].dataPtr() + a_offset;
+        m_uy = attribs[PIdx::uz].dataPtr() + a_offset;
+        m_uz = attribs[PIdx::uz].dataPtr() + a_offset;
```

Comment:
Little copy-paste bug:
```suggestion
        m_ux = attribs[PIdx::ux].dataPtr() + a_offset;
        m_uy = attribs[PIdx::uy].dataPtr() + a_offset;
        m_uz = attribs[PIdx::uz].dataPtr() + a_offset;
```